CHANGELOG.md

```diff
@@ -62,6 +62,15 @@ New Features
   truth information.  cf. demos 9 and 10. (#301, #691)
 - Added methods calculateHLR, calculateMomentRadius, and calculateFWHM to both
   GSObject and Image. (#308)
+- Added LookupTable2D to facilitate quick interpolation of two-dimensional
+  tabular data. (#465)
+- Added AtmosphericPhaseScreen, OpticalPhaseScreen, and PhaseScreenList used
+  to generate PSFs via Fourier optics. (#549)
+- Added PhaseScreenPSF to transform PhaseScreens into GSObjects.  (#549)
+- Added Atmosphere function to conveniently assemble a multi-layer atmosphere
+  PhaseScreenList. (#549)
+- Rewrote OpticalPSF to operate under the PhaseScreen framework to enable
```

Comment:
>2) I understand why you're not adding this to a demo (too slow) but in the list of demos in README, perhaps you could mention the script that demonstrates this capability?

With the current (arbitrary) default command line arguments (4m aperture, 3 layers, 30ms time_step, 3 second exptime, ...), the script takes about 3min on my laptop.  If one was just computing the final PSF, and not also rendering all of the instantaneous PSFs and wavefronts, then I bet it would be quite a bit faster, so maybe not *too* slow.  (Though it's not nearly as entertaining to get a noisy image of a single star than to see the movie with both the PSF and wavefront, of course.)  Maybe we should at least move the script from /devel/modules/phase_psf/ to /examples/?

Also, this script is super useful!  I just noticed that my screens unexpectedly had *acceleration* in them (grr... stupid error with += combined with @property return values are references not copies...), so my nice breezy 5m/s wind at the start of an exposure was becoming a supersonic 500 m/s monstrosity after 3 seconds.  Looks like I'll have to redo the accuracy estimates...  :/  At least I think I know why now I couldn't see an correlations in recently generated PSF fields.